specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2859 @@
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomResult"
+        }
+      ],
+      "required": [
+        "documents"
+      ]
+    },
+    "SingleClassificationDocumentResult": {
+      "type": "object",
+      "properties": {
+        "classification": {
+          "$ref": "#/definitions/ClassificationResult"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentResult"
+        }
+      ],
+      "required": [
+        "classification"
+      ]
+    },
+    "ClassificationResult": {
+      "type": "object",
+      "required": [
+        "category",
+        "confidenceScore"
+      ],
+      "properties": {
+        "category": {
+          "type": "string",
+          "description": "Classification type."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "Confidence score between 0 and 1 of the recognized classification."
+        }
+      }
+    },
+    "CustomMultiClassificationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Custom Multi Classification task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomTaskParameters"
+        }
+      ]
+    },
+    "CustomMultiClassificationLROTask": {
+      "type": "object",
+      "description": "Use custom models to classify text into multi label taxonomy",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/CustomMultiClassificationTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "CustomMultiClassification"
+    },
+    "CustomMultiClassificationResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Response by document",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/MultiClassificationDocumentResult"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomResult"
+        }
+      ],
+      "required": [
+        "documents"
+      ]
+    },
+    "MultiClassificationDocumentResult": {
+      "type": "object",
+      "properties": {
+        "classifications": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClassificationResult"
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentResult"
+        }
+      ],
+      "required": [
+        "classifications"
+      ]
+    },
+    "HealthcareTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Healthcare task.",
+      "properties": {
+        "structureFHIR": {
+          "type": "boolean",
+          "default": false
+        },
```

Comment:
It seems FHIR only appear in this request and its response. Does it at least need a "description" about what it does?

Reply:
added a description to both